sled-agent/src/services.rs

```diff
@@ -613,13 +645,21 @@ impl ServiceManager {
             // opte_ports=
             vec![],
             bootstrap_vnic,
-            link,
+            links.clone(),
```

Comment:
Regarding my comment above about `derive(Clone)` - I don't think you need to clone the links here. Cloning their names would be totally reasonable, so you can iterate on 'em when the zone is up, but "ownership" of the links should be given to the "zone" object being created.

Reply:
Addressed in 8a37b046